stable/jenkins/templates/config.yaml

```diff
@@ -69,7 +69,15 @@ data:
               </containers>
               <envVars/>
               <annotations/>
+{{- if .Values.Agent.ImagePullSecret }}
+              <imagePullSecrets>
+                <org.csanchez.jenkins.plugins.kubernetes.PodImagePullSecret>
+                  <name>{{ .Values.Agent.ImagePullSecret }}</name>
+                </org.csanchez.jenkins.plugins.kubernetes.PodImagePullSecret>
+              </imagePullSecrets>
```

Comment:
It's correct, as the L79 close is a single line close.  Basically, I can remove it if I do as you indicate in the first comment.